app/jobs/renewal_job.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class RenewalJob < ApplicationJob
+  queue_as :default
+
+  RENEWAL_CUSTOM_MESSAGELIST = {
+    "hatErrorResponse.141": 'Renewal limit reached, cannot be renewed.',
+    "hatErrorResponse.7703": 'Renewal limit reached, cannot be renewed.',
+    "hatErrorResponse.105": 'Item has been recalled, cannot be renewed.',
+    "hatErrorResponse.252": 'Item has holds, cannot be renewed.',
+    "hatErrorResponse.46": 'Item on reserve, cannot be renewed.',
+    "unhandledException": ''
+  }.with_indifferent_access
+
+  def perform(resource:, item_key:, session_token:)
+    symphony_client = SymphonyClient.new
+    redis_client = Redis.new
```

Comment:
This Job can change to mimic https://github.com/psu-libraries/myaccount/pull/194/files#diff-949384a9b4789b027b8340913bfe9858 in terms of the way the connection to Redis is handled.

Reply:
Yes,I was waiting on the PR to be merged, now that it is I will update :)

Reply:
updated

Reply:
Thanks, looks great